keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/MultivariateGaussianVertex.java

```diff
@@ -72,7 +72,7 @@ public MultivariateGaussianVertex(DoubleVertex mu, double covariance) {
     }
 
     public MultivariateGaussianVertex(double mu, double covariance) {
-        this(oneByOneMatrix(mu), oneByOneMatrix(covariance));
+        this(new ConstantDoubleVertex(new double[]{mu}), oneByOneMatrix(covariance));
```

Comment:
I wonder if we should have a `DoubleTensor.vector` factory method, alongside the existing `DoubleTensor.scalar`. I don't find it obvious looking at `new double[]{mu}` to know that it is a vector.

Reply:
👍